apps/web/src/containers/LandingPageView/components/our-backers/components/user-cards.tsx

```diff
@@ -0,0 +1,110 @@
+import { Box, styled, useTheme } from '@mui/material';
+import { useUserCards } from '../use-user-cards';
+import { colors } from '@notional-finance/styles';
+import { BodySecondary, ExternalLink } from '@notional-finance/mui';
+import twitter from '@notional-finance/assets/images/logos/twitter.svg';
+import quotes from '@notional-finance/assets/images/logos/quotes.svg';
+
+export const UserCards = () => {
+  const theme = useTheme();
+  const userCards = useUserCards();
+  return (
+    <CardContainer>
+      {userCards.map(({ name, title, text, twitterLink, logo }, index) => (
+        <Wrapper key={index}>
+          <img
+            src={logo}
+            alt="logo"
+            style={{
+              height: theme.spacing(10),
+              width: '100%',
+              marginTop: `-${theme.spacing(4)}`,
+            }}
+          />
+          <Card key={index}>
+            <Box>
+              <img src={quotes} alt="quotes" style={{ height: '31px' }} />
+              <BodyText sx={{ marginTop: theme.spacing(3) }}>{text}</BodyText>
+            </Box>
+            <TitleContainer>
+              <div>
+                <BodyText sx={{ color: colors.neonTurquoise }}>{name}</BodyText>
+                <BodyText
+                  sx={{
+                    color: colors.neonTurquoise,
+                    fontStyle: 'italic',
+                  }}
+                >
+                  {title}
+                </BodyText>
+              </div>
+              <ExternalLink href={twitterLink}>
+                <img src={twitter} alt="twitter" style={{ height: '20px' }} />
+              </ExternalLink>
+            </TitleContainer>
+          </Card>
+        </Wrapper>
+      ))}
+    </CardContainer>
+  );
+};
+
+const CardContainer = styled(Box)(
+  ({ theme }) => `
+        width: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        flex-direction: row;
+        gap: ${theme.spacing(6)};
+        ${theme.breakpoints.down(1220)} {
```

Comment:
Similar question on breakpoint sizing here.

Reply:
Across the landing page I have been using 1220 and 1000 custom breakpoints because our predefined ones in the theme did not work with the design.

Reply:
Ok, can we just add them into the theme object like "sm-landing" and "md-landing" or something like that so they are labeled in the components?

Reply:
Sure